charts/flink-job/templates/servicemonitor.yaml

```
@@ -10,6 +10,7 @@ metadata:
     {{- toYaml . | nindent 4 }}
     {{- end}}
 spec:
+  jobLabel: {{ .jobLabel }}
```

Just to make sure that someone doesn't use labels with weird-looking types
```suggestion
  jobLabel: {{ .jobLabel | quote }}
```